diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/BooleanExpressionsRule.kt

```diff
@@ -142,6 +144,13 @@ class BooleanExpressionsRule(configRules: List<RulesConfig>) : DiktatRule(
             operationReferenceText == "&&" || operationReferenceText == "||"
         }
 
+    private fun ASTNode.textWithoutComments() = findAllNodesWithCondition(withSelf = false, condition = {
```

Comment:
why do you need to mention arguments explicitly?

Reply:
Because we somehow have `condition` as first argument and `withSelf` as second. I wanted to have usual lambda syntax withput parentheses, but apparently you can't use it even with named arguments. I guess I can just change the function's signature

Reply:
Fixed